app/views/users/new.html.erb

```diff
@@ -1,24 +1,8 @@
 <% provide :title, (t "static_pages.home.sign_up") %>
+<% provide :button_text, (t "static_pages.crea_acco") %>
 <h1><%= t "static_pages.home.sign_up" %></h1>
-
 <div class="row">
   <div class="col-md-6 col-md-offset-3">
-    <%= form_for @user do |f| %>
-      <%= render "shared/error_messages", user: @user %>
-
-      <%= f.label :name %>
-      <%= f.text_field :name, class: "form-control" %>
-
-      <%= f.label :email %>
-      <%= f.email_field :email, class: "form-control" %>
-
-      <%= f.label :password %>
-      <%= f.password_field :password, class: "form-control" %>
-
-      <%= f.label :password_confirmation, (t "static_pages.confirmation") %>
-      <%= f.password_field :password_confirmation, class: "form-control" %>
-
-      <%= f.submit (t "static_pages.crea_acco"), class: "btn btn-primary" %>
-    <% end %>
+    <%= render "form", user: @user %>
```

Comment:
đây dùng "form"
lsao mình biết được form nào là new form nào là edit?

Reply:
Em thấy phần form này của edit và new là hoàn toàn giống nhau
Chỉ khác là sau khi submit form này thì 2 request khác nhau patch và post của edit và new thôi ạ